ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/js/wizardview.js

```diff
@@ -290,7 +290,14 @@
             var activeTabElement = this.#getCachedTabs()[activeIndex];
             var activeChildId = activeTabElement.id.substring(0, activeTabElement.id.lastIndexOf(Wizard.#tabIdSuffix));
             var activeChildView = this.getChild(activeChildId);
-            var validationErrorList = activeChildView.getModel().validate();
```

Comment:
Why are we still using var in this code ?

Reply:
Please add test case for the use-case

Reply:
yeah I'm not sure why the whole file doesn't use any ES6. As for test case, the use case is for Compare forms from formsmanager, so will add it wherever we add tests for versioning, copare etc.